bazel/linux/defs.bzl

```diff
@@ -237,17 +239,19 @@ def kernel_module(*args, **kwargs):
       kernels: list of kernel (same as above). kernel_module will instantiate multiple
             kernel_module_rule, one per kernel, and ensure they all build in parallel.
     """
+    if "makefile" not in kwargs:
+        kwargs["makefile"] = "Makefile"
+
     if "srcs" not in kwargs:
-        kwargs["srcs"] = native.glob(include = ["**/*"], exclude = BUILD_LEFTOVERS, allow_empty = False)
+        include = ["**/*.c", "**/*.h", kwargs["makefile"]]
+        exclude = BUILD_LEFTOVERS + ["**/*.mod.c"]
```

Comment:
Can you fix BUILD_LEFTOVERS instead? Eg, in theory *.mod.c is already in there? It's probably not working either because of the missing "**/" or the "[co]" argument.

Reply:
Are you okay with the changes to the include pattern? Or do you prefer to keep it `**/*`?